src/core_ocean/analysis_members/mpas_ocn_lagrangian_particle_tracking.F

```diff
@@ -1939,20 +2008,39 @@ subroutine initialize_particle_properties(domain, timeLevel, err)!{{{
           !}}}
           if ( associated(activeTracers) ) then
             if (config_AM_lagrPartTrack_sample_temperature) then
-             !call mpas_log_write(' testing output for temperature')
+             call mpas_log_write(' testing output for temperature')
              call interp_cell_scalars(iLevel, maxLevelCell(iCell), -zLevelParticle, -zMid(:,iCell), &
-               activeTracers(index_temperature,:,iCell), temperature)
-             !call mpas_log_write('done testing output for temperature')
+               activeTracers(index_temperature,:,iCell), particleTemperature)
+             call mpas_log_write('done testing output for temperature')
             end if
 
             if (config_AM_lagrPartTrack_sample_salinity) then
-             !call mpas_log_write(' testing output for salinity')
+             call mpas_log_write(' testing output for salinity')
              call interp_cell_scalars(iLevel, maxLevelCell(iCell), -zLevelParticle, -zMid(:,iCell), &
-               activeTracers(index_salinity,:,iCell), salinity)
-             !call mpas_log_write('done testing output for salinity')
+               activeTracers(index_salinity,:,iCell), particleSalinity)
+             call mpas_log_write('done testing output for salinity')
             end if
           end if
 
+          if (config_use_ecosysTracers) then
+            call mpas_pool_get_array(tracersPool, 'ecosysTracers', ecosysTracers, timeLevel)
+            if ( associated(ecosysTracers) ) then
+              if (config_AM_lagrPartTrack_sample_DIC) then
+                call mpas_log_write('testing output for DIC')
+                call interp_cell_scalars(iLevel, maxLevelCell(iCell), -zLevelParticle, -zMid(:,iCell), &
+                  ecosysTracers(index_DIC,:,iCell), particleDIC)
```

Comment:
Just to check here: is `index_DIC` defined?  Sometimes this can cause a seg-fault.  Just something to quickly check.

Reply:
@pwolfram, yes see lines 324-325 and 1838-1839. index_DIC and index_ALK are defined just below index_temperature and index_salinity.